js/__tests__/bbs/credentials-v2.test.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable max-len */
+const {
+  Bls12381G2KeyPair,
+  BbsBlsSignature2020,
+  BbsBlsSignatureProof2020,
+  deriveProof,
+} = require('@mattrglobal/jsonld-signatures-bbs');
+
+const fs = require('fs');
+const path = require('path');
+const vcjs = require('@transmute/vc.js');
+
+const vcld = vcjs.ld;
+
+const didKeyFixture = require('../../__fixtures__/did-key-bls12381.json');
+
+const k0 = didKeyFixture[0].g2['application/did+ld+json'];
+k0.id = k0.controller + k0.id;
+
+const credentialTemplate = require('../../__fixtures__/credentials/bbs/credentialTemplate.json');
+let verifiableCredential;
+
+const documentLoader = url => {
```

Comment:
is this possible to encapsulate into a common customDocumentLoader file rather than redefining frequently?

Reply:
yes, I think its probably wise to do that now.

Reply:
done